### ISSUE TITLE
Cannot export keras Sequential model to ONNX format

### ISSUE THREAD

**Issue by ladodc** [onnx/keras-onnx#519]
Hallo, I have a problem with exporting keras model to ONNX format. 
I always get following error on this line onnx_model = keras2onnx.convert_keras(nn):
_AttributeError: 'tuple' object has no attribute 'layer'_
My versions are 
_Python: 3.7
keras: 2.3.1
tensorflow: 2.1.0
onnx: 1.7.0_

**Here is my code:**
`​from keras import models
from keras import layers
import keras
from keras.datasets import mnist

(train_images, train_labels), (test_images, test_labels) = mnist.load_data()

train_images = train_images.reshape((60000, 28 * 28))

train_images = train_images.astype('float32') / 255

test_images = test_images.reshape((10000, 28 * 28))
test_images = test_images.astype('float32') / 255

from keras.utils import to_categorical

train_labels = to_categorical(train_labels)
test_labels = to_categorical(test_labels)

nn = models.Sequential()
nn.add(layers.Dense(512, activation='relu', input_shape=(28 * 28,)))
nn.add(layers.Dense(10, activation='softmax'))

nn.compile(optimizer='rmsprop',
                loss='categorical_crossentropy',
                metrics=['accuracy'])

nn.fit(train_images, train_labels, epochs=5, batch_size=128)

import keras2onnx

onnx_model = keras2onnx.convert_keras(nn)

**Comment by jiafatom** [comment]
Since you installed tensorflow 2.x, the converter default behavior is to treat the model as tf.keras model. Seems that your model is essentially a keras standalone model, so you need overwrite the default behavior by setting env variable `TF_KERAS=0` before importing keras2onnx.

**Comment by ladodc** [comment]
Hi jiafatom, 
thank you for your response. I have tried your suggestion but with no result. But I have also tried implemented keras API in tensorflow tf.keras and then it worked. So your intention is right.  Back to standalone keras solution. I have set TF_KERAS=0 like this. 
```
import os
os.environ['TF_KERAS'] = '0'
```
But with no effect. It looks like my setting has no effect. Is this the correct way how to set up TF_KERAS=0
